pkgs/flox-manpages/default.nix

```diff
@@ -47,6 +47,8 @@ in
 
 
     fd ".*\.md" -d 1 ./ -x ${compileManPageBin} {} $buildDir/{/.}.1
+    rm -f $buildDir/manifest.toml.1
+    ${compileManPageBin} manifest.toml.md $buildDir/manifest.toml.5
```

Comment:
nonblocking nit: how come not just `mv`?

Reply:
Hmmm ... good question. I had it in my head that the man page compiler was somehow embedding the section number in the content but of course that's not governed by the filename. I've tested and it does the right thing - will change!